core/src/main/java/hudson/model/Descriptor.java

```diff
@@ -77,6 +77,7 @@
 import java.util.IdentityHashMap;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
```

Comment:
@oleg-nenashev Should we be using a FindBugs annotation here as per @stephenc's licensing comments?